index.js

```diff
@@ -1,9 +1,9 @@
 const express = require('express')
 const bodyParser = require('body-parser')
 const cookieParser = require("cookie-parser")
-var cors = require('cors')
 const sessions = require('express-session')
 const db = require("./src/config/database.config.js")
+const db_stu = require("./src/config/database.config.js")
```

Comment:
Why twice the same file and same database ?
If planning on maintaining two seperate databases for admin and student details then it can be done in just one database.